.travis.yml

```diff
@@ -1,5 +1,9 @@
+dist: xenial
 language: node_js
 node_js:
-  - "9"
+  - "10"
```

Comment:
I prefer to add also `"node"` to check in the latest node version

Reply:
yes, i will add that instead of 10